src/xdocs/google_style.xml

```diff
@@ -67,19 +56,29 @@
       <subsection name="Coverage table" id="Google_Coverage_table">
         <p>
           <b>ATTENTION:</b>
-          Links to config and test in following table reference to latest (not released yet) config.
+          Links to config in following table reference to latest (not released yet) config.
```

Comment:
This goes with samples too. They are links to master, which is unreleased.

Reply:
fixed